scripts/image-generator/imageGenerator.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-console */
+const ChessImageGenerator = require('chess-image-generator');
+const fs = require('fs');
+const path = require('path');
+
+const PGN_FOLDER = './pgn';
+const PNG_FOLDER = './png';
+const BASE64_FOLDER = './base64';
+
+/**
+ * This is a temporary script to generate Chess images given a folder of PGN files
+ * TODO: Convert image generator script into a backend endpoint
+ */
+const generatePNGImages = async () => {
+  try {
+    const pgnFilepaths = fs.readdirSync(PGN_FOLDER);
+    pgnFilepaths.forEach(async (pgnFilepath) => {
+      const filename = path.basename(pgnFilepath, path.extname(pgnFilepath));
+      const pgnFile = fs.readFileSync(`${PGN_FOLDER}/${pgnFilepath}`, 'utf8');
+      const imageGenerator = new ChessImageGenerator();
+      await imageGenerator.loadPGN(pgnFile);
+      const outputFilepath = `./${PNG_FOLDER}/${filename}.png`;
+      await imageGenerator.generatePNG(outputFilepath);
+    });
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+const generateBase64Images = async () => {
```

Comment:
I assume this function will eventually be used to generate Base64 images that will be stored in the database?  I was thinking it might be better to generate PNG images using the PGN like what you did with `generatePGNImages` when the images are needed rather than storing base64 images for approximately 1327 opening positions.

Reply:
This doesn't really affect the work you did for #30 but just wanted to bring it up.

Reply:
Yeah, that's a good thought! I'm not too sure what the best way to store it, so this is something we should definitely talk about later on. :)